st2api/st2api/listener.py

```diff
@@ -49,6 +44,11 @@ def __init__(self, connection):
 
     def get_consumers(self, consumer, channel):
         return [
+            consumer(queues=[announcement.get_queue(routing_key=publishers.ANY_RK,
```

Comment:
actually, no. The exclusive means it would be deleted as soon as consumer stop listening it. Since we want every consumer to listen to its own queue instead of sharing the same queue with another consumer, we absolutely can not give it a name.

The problem with sharing is that if one consumer will consume the message, another one will never get it so if one st2api instance get an update on execution, the users from another instance of st2api will never get it.